scss/styles.scss

```diff
@@ -3,3 +3,13 @@
 * {
   font-family: 'Roboto', sans-serif;
 }
+
+.flex-body {
+  display: flex;
+  flex-direction: column;
+
+  main {
+    margin: 0 auto;
+    width: 90%;
```

Comment:
I think you should set width: 100% and a max-width in pixels to define the container's size. using 90% generates different spacing and widths depending on the screen and it's better to keep at a max size instead and use padding to generate this separation from the borders.